test/export/test_export.py

```diff
@@ -2262,17 +2229,19 @@ def forward(self, x, y):
         self.assertTrue(torch.allclose(ep.module()(*test_inp), fn(*test_inp)))
 
     @testing.expectedFailureNonStrict
+    @testing.expectedFailureRetraceability
```

Comment:
Could you link the task that you filed for this?

Reply:
Might be related to other tasks, there's an error message diff:
> AssertionError: "is outside of inline constraint" does not match "Runtime assertion failed for _local_scalar_dense_default >= 2"